scan-command/src/main/java/io/ballerina/scan/internal/CoreRule.java

```diff
@@ -32,7 +32,8 @@
 enum CoreRule {
     AVOID_CHECKPANIC(RuleFactory.createRule(1, "Avoid checkpanic", RuleKind.CODE_SMELL)),
     UNUSED_FUNCTION_PARAMETER(RuleFactory.createRule(2,
-            "Unused function parameter", RuleKind.CODE_SMELL));
+            "Unused function parameter", RuleKind.CODE_SMELL)),
+    INVALID_RANGE_OPERATOR(RuleFactory.createRule(10, "Invalid range operator", RuleKind.CODE_SMELL));
```

Comment:
Shall we use expression instead of operator? Let's try and stick to spec terms wherever possible (where they also are clear to the user).